app/lib/features/home/pages/home_page.dart

```diff
@@ -40,6 +41,7 @@ class _HomePageState extends ConsumerState<HomePage> {
   void initState() {
     super.initState();
     pageController = PageController(initialPage: _selectedIndex);
+    screenshotController = ScreenshotController();
```

Comment:
Is this really needed to be constructed here? ```late``` is bad if for some reason widget doesn't get inserted it would throw ```InitializationError```.
How about having it initialised above?

Reply:
I thought it is reasonable to initialize `pageController` and `screenshotController` at the same place.
Your idea is great, because `screenshotController` needs no arguments.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/522/commits/0f4fee6a5f70b48381102fcf28602f0e690e86d1